.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 
 language: node_js
 node_js:
-  - "12.14.0"
+  - "14.15.5"
```

Comment:
I would use the latest version of Node: 14.18.1

Reply:
updated